packages/block-editor/src/components/url-input/index.js

```diff
@@ -134,7 +134,22 @@ class URLInput extends Component {
 			return;
 		}
 
-		const isInitialSuggestions = ! ( value && value.length );
+		// Test search string for:
+		// 1. only whitespace characters (via inversion of test for any non-whitespace character).
+		// 2. must have a length - otherwise test from #1 will be truthy for string with no characters.
+		const containsOnlyWhitespace = !! (
+			! /\S/.test( value ) && value?.length
+		);
```

Comment:
How about:
```suggestion
		const containsOnlyWhitespace = ! value?.length || value.trim() === '';
```

Reply:
Yeh actually that's a lot better 🤣 Thanks Adam!